Week1/project/index.js

```diff
@@ -1 +1,38 @@
-// your code goes in here
\ No newline at end of file
+const quotes = [{
+  quote: 'Wisely, and slow.They stumble that run fast.',
+  by: ' William Shakespeare'
+}, {
+  quote: 'If you judge people, you have no time to love them.',
+  by: 'Mother Teresa'
+},
+{
+  quote: ' All that we are is the result of what we have thought.',
+  by: 'Buddha'
+},
+{
+  quote: 'The future belongs to those who prepare for it today.',
+  by: 'Malcolm X'
+},
+{
+  quote: 'It always seems impossible until it’s done.',
+  by: 'Nelson Mandela'
+},
+{
+  quote: 'I don’t think of all the misery but of the beauty that still remains.',
+  by: 'Anne Frank'
+}
+]
+const button = document.querySelector('button');
+const h1 = document.querySelector('h1');
+const h2 = document.querySelector('h2');
+console.log(button);
+
+window.addEventListener('load', generateQuote);
+
+function generateQuote() {
+  h1.innerHTML = quotes[random()].quote;
+  h2.innerHTML = quotes[random()].by;
+}
+
+const random = () => { return Math.round(Math.random(quotes.length)) };
```

Comment:
Minor comment: you can leave out the curly brackets and the return if you write an arrow function on one line:
```suggestion
const random = () => Math.round(Math.random(quotes.length));
```
or return the element directly:
```suggestion
const randomQuote = () => quotes[ Math.round(Math.random(quotes.length)) ];
```

Reply:
@bnhoogendijk Thank you very much for your comments. I have updated my code based on your suggestions.